Jenkinsfile

```diff
@@ -0,0 +1,76 @@
+@Library('jenkins.shared.library') _
+
+environment {
+  HELM_IMAGE = "infoblox/helm:3.2.4-5b243a2"
+  REGISTRY = "core-harbor-prod.sdp.infoblox.com"
+  TAG = sh(script: "git describe --always --long --tags", returnStdout: true).trim()
```

Comment:
This is an edge case, but if hypothetically speaking the "Package Chart" or "Push Chart" stages were to error out, then  if you tried to repeat the merge build without source code changes then the "Push Image" stage would fail because of Harbor's immutable tag feature.   Appending the Jenkins build number to the tag would resolve that case. Something like:
```
IMAGE_TAG="${TAG}-j${BUILD_NUMBER}"
```
The other alternative for this use case is either push some innocuous change to force a rebuild, or use Jenkins' Replay feature and manually comment out the `signDockerImage()` call

Reply:
Or wrap `signDockerImage()` in `try`/`catch`... I'll add one of these options.